package.json

```diff
@@ -2,7 +2,7 @@
   "name": "gather",
   "displayName": "Gather",
   "description": "Adds gather functionality to the Python Extension",
-  "version": "2020.7.1",
+  "version": "2020.11.0",
   "publisher": "ms-python",
   "author": {
     "name": "Microsoft Corporation"
```

Comment:
Homepage might need update python->jupyter as well when public.

Reply:
You mean the publisher, right?

Reply:
No, there is also a homepage like to the vscode-python github.